fecgov/fec-eregs
# Feature/458 upgrade dev dependencies

# Summary

- Addressing security concerns in devDependencies.
- Resolves #458 

# Changes

- Updated the owner and license information
- Updated the version of jQuery from `3.2.1` => `3.4.1`
- Updating Grunt from `0.4.5` => `1.0.4` also gave us:

package | from | to
:--- | ---: | ---:
handlebars | `4.0.111` | `4.5.1`
underscore.string | `2.2.1` | `3.3.5`
js-yaml | `2.0.5` | `3.13.1`
tunnel-agent | `0.4.3` | `0.6.0`
extend | `3.0.1` | `3.0.2`
cached-path-relative | `1.0`.x | `1.0.2`
lodash | `2.4.2` | `4.17.10`
lodash.mergewith | removed | removed
is-my-json-valid | removed | removed

After the updates, the 25+ errors reported via `npm audit` dropped to 0.

## How to Test
I think...

- Get eregs running locally
- `npm i`
- ☝️ should throw no errors
- `npm run build`
- ☝️ should throw no errors
- Check the site to make sure it works?

I'd like to check on dev and stage just to make sure. These were devDependencies so should have thrown errors on `npm run build` if they were going to.
